rust/xaynet-server/src/state_machine/phases/idle.rs

```diff
@@ -33,47 +31,49 @@ pub enum IdleStateError {
 pub struct Idle;
 
 #[async_trait]
-impl<S> Phase<S> for PhaseState<Idle, S>
+impl<T> Phase<T> for PhaseState<Idle, T>
 where
-    S: Storage,
+    T: Storage,
 {
     const NAME: PhaseName = PhaseName::Idle;
 
-    async fn run(&mut self) -> Result<(), PhaseStateError> {
+    async fn process(&mut self) -> Result<(), PhaseStateError> {
+        info!("removing phase dictionaries from previous round");
```

Comment:
the invalidation of the dicts is missing here
```suggestion
        info!("removing phase dictionaries from previous round");
        info!("broadcasting invalidation of sum dictionary from previous round");
        events.broadcast_sum_dict(DictionaryUpdate::Invalidate);

        info!("broadcasting invalidation of seed dictionary from previous round");
        events.broadcast_seed_dict(DictionaryUpdate::Invalidate);
```

Reply:
i somewhat forgot about a successful round of the state machine :D in case of an error the invalidation is now handled in the error phase. in case of a successful round, i think the earliest we could invalidate the sum and seed dicts is after the sum2 phase, that means in `broadcast()` of the sum2 phase. we should probably add that there, wdyt?

Reply:
I like the idea to add this to the broadcast method of the sum2 phase 👍